server/src/test/java/org/apache/iotdb/db/engine/merge/MergeOverLapTest.java

```diff
@@ -29,6 +29,7 @@
 import java.util.List;
 import org.apache.commons.io.FileUtils;
 import org.apache.iotdb.db.conf.IoTDBConstant;
+import org.apache.iotdb.db.conf.IoTDBDescriptor;
```

Comment:
This import is added while there are no other changes to this file, please take a look.

Reply:
fixed~